### DISCUSSION TITLE
external token before hitting strapi api ?

### DISCUSSION THREAD

**Discussion by dunods** [strapi/strapi#6087] [Legacy Topics]
helo everyone, 

is there anyone experience to use external authorization token before hitting the api?

for example:

user login to localhost:3000/login **(not strapi api /external)** and successfully receive the token ? and we have strapi api at localhost:1335/products, but we to use that token we receive before to authorise to that strapi endpoint before retrieving the data...

any clue ?

**Comment by robertpiosik** [comment]
Hmm, I would create custom policy on the endpoint `/products` where I would verify that token.
